### PR TITLE
doc: Fix Image 'set_pixel' doc for use of 'create'

### LINKED ISSUE #72904: [rc1] Image creation doesn't work
### Godot version

4.0 rc1

### System information

Linux Manjaro, Intel Kaby Lake

### Issue description

```
var img = Image.new()
img.create(1000, 1000, false, Image.FORMAT_RGBA8)
	
print("Img size: ", img.get_size())
```

size is (0,0) and obviously set_pixel spams errors. Looked at docs both in editor and on website, no indication of any changes.

### Steps to reproduce

See above

### Minimal reproduction project

N/A

### PR DESCRIPTION
**Pull request by akien-mga**
Fixes #72904.

Should it stay as is for C# @raulsntos? Or does it support the static method?

### CODE REVIEW

**Comment by raulsntos** [comment]
The `Create` method is also static in C#, leaving it as is would result in a compiler error. So yeah, it also needs to be updated in the C# example.

File: doc/classes/Image.xml [hunk 1]
```diff
@@ -510,8 +509,7 @@
 				[gdscript]
 				var img_width = 10
 				var img_height = 5
-				var img = Image.new()
-				img.create(img_width, img_height, false, Image.FORMAT_RGBA8)
+				var img = Image.Create(img_width, img_height, false, Image.FORMAT_RGBA8)
```
Comment (raulsntos) [inline doc/classes/Image.xml:512]: This is GDScript.

```suggestion
				var img = Image.create(img_width, img_height, false, Image.FORMAT_RGBA8)
```

**Comment by akien-mga** [comment]
What's the syntax?

**Comment by raulsntos** [comment]
> What's the syntax?

```csharp
var img = Image.Create(imgWidth, imgHeight, false, Image.Format.Rgba8);
```

**Review by raulsntos — APPROVED** [review]

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**